### PR TITLE
Fixes error returning photo proof details

### LINKED ISSUE #11: Photo Proof details not being returned
PhotoProofDetails was returning nil rather than the details. This is probably causing the errors on the live server.

### PR DESCRIPTION
**Pull request by ilynne**
Photo proof details were not being returned correctly. This should fix that error.

Also adds pry gem.

closes #11

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -51,6 +51,7 @@ group :development, :test do
   gem 'launchy'
   gem 'database_cleaner', '>= 1.1.1'
   gem 'letter_opener'
+  gem 'pry'
```
Comment (becker1) [inline Gemfile]: Do we need `pry` in test too or is it just for development?

Comment (ilynne) [inline Gemfile]: Good question -- I see it not restricted at all on mykotis (bad example to follow, I know) and showcase-admin. I do use it in test, but I can't see a good reason for it in production.

Comment (jayfredlund) [inline Gemfile]: It should be development and test.  Not production.

**Comment by jayfredlund** [comment]
In your description, you shouldn't say "closes" with the PR number. :)  (Nothing to do related to this comment)

**Pull request merged**